migrations/20160216120159_create_initial_schema.js

```diff
@@ -51,7 +51,7 @@ exports.up = (knex) => {
       .notNullable();
     table.text('study_design')
       .notNullable();
-    table.text('study_phase')
+    table.jsonb('study_phase')
```

Comment:
Why you're using `jsonb` instead of an actual array type (maybe even an array of enum)?